blog/2024-01-24-embed-figma/index.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Embed Figma in Stately
+description: You can now embed Figma frames in Stately states
+tags: [stately, studio, figma, design, statecharts, state machines, xstate]
+authors: [nick]
+date: 2024-01-24
+slug: 2024-01-24-embed-figma
+image: /blog/2024-01-24-embed-figma.png
+---
+
+We already kicked off 2024 with several major updates to Stately such as [inspecting live XState code](/blog/2024-01-15-stately-inspector), [syncing with GitHub](/blog/2024-01-09-introducing-bidirectional-github-sync), and [writing code for actions and actors in the Studio](/blog/2024-01-09-introducing-sources). Now we’re excited to announce that we’ve also launched an integration with Figma.
+
+You can now attach frames in Figma to states in Stately.
+
+<p>
+  <ThemedImage
+    alt="Three Figma frames displayed in three Stately states for a music player. The ready state shows a record stack app view, following the openAlbum event, the viewingAlbum state shows the Wild Winds album coverart and information, from the viewingAlbum state, there’s a play event that takes you to the playingAlbum state which shows the album list for the Wild Winds album and a player progress bar. The pause event returns from the playingAlbum state to the viewingAlbum state."
+    sources={{
+      light: '/blog/2024-01-24-embed-figma/figma-light.png',
+      dark: '/blog/2024-01-24-embed-figma/figma-dark.png',
+    }}
+  />
+</p>
+
+## Improve your team workflows by combining Stately and Figma
+
+Embedding Figma into Stately can significantly streamline and enhance a team’s workflow.
+
+Being able to view mockups of the states you are discussing make it easy to _see_ what might be missing and identify potential improvements. It allows team members to view designs directly as part of the app logic itself.
+
+Two examples of how this can work are:
+
+- Engineers can import state machine code, then designers can fill out images for each state of the actual logic in the code.
+
+- Designers, PMs, and other stakeholders can create the expected behavior by embedding Figma frames into a Stately diagram, then engineers can continue adding the functionality from the automatically scaffolded code.
+
+This seamless integration can speed up the design process and ensure all team members are on the same page. Team members can easily share design details, run critiques, and iterate with people across different specializations.
+
+It also ensures that everyone is always looking at the most up-to-date design, as changes made in Figma are instantly reflected in the embedded view. And with [GitHub Sync](blog/2024-01-09-introducing-bidirectional-github-sync) enabled, changes to the code can even automatically update how the Figma frames are structured in Stately.
```

Comment:
```suggestion
It also ensures that everyone is always looking at the most up-to-date design, as changes made in Figma are instantly reflected in the embedded view. And with [GitHub Sync](blog/2024-01-09-introducing-bidirectional-github-sync) enabled, changes to the code can even automatically update where the Figma frames are displayed in Stately.
```

Just because otherwise it sounds a bit like you can change the content of the Figma frames with code.